hazelcast/src/main/java/com/hazelcast/spi/Operation.java

```diff
@@ -97,7 +100,21 @@ public void beforeRun() throws Exception {
     }
 
     // runs after wait-support, supposed to do actual operation
-    public abstract void run() throws Exception;
+    @Deprecated
+    public void run() throws Exception {
+    }
+
+    public CallStatus call() throws Exception {
```

Comment:
The magic parts where the old Operation.run keeps on working, but the new call functionality can be used as well.